### ISSUE TITLE
Ileal stricture

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11201]
**Preferred term label:**
Ileal stricture

**Synonyms**
Stricture of ileum

**Definition (free text, please give PubMed ID)**
A pathological narrowing of the ileum that is caused by the development of a ring of scar tissue that constricts the ileal lumen.
PMID:31476299
PMID:34400338
PMID:12415093

Comment:
The differential diagnosis of ileal stricture includes tuberculosis, Crohn’s disease, pelvic inflammation, ischemia, radiation enteritis, carcinoid infiltration, lymphoma, and diffuse enteropathy-that is, disorders where there is inflammation, infiltration, or edema of the small bowel.

**Parent term (use hpo.jax.org/app)**
Abnormal ileum morphology HP:0001549

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619079

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
